docs/index.md

```diff
@@ -92,8 +92,9 @@ you can install via pip, conda-forge, or from source.
 
 ### From pip, with "batteries included"
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
-3.8-3.10 using pip:
+napari can be installed on most macOS, Linux, and Windows systems with
+ [![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org) by
+ using pip:
```

Comment:
```suggestion
napari can be installed on most macOS, Linux, and Windows systems with
 ![Supported Python versions](https://img.shields.io/pypi/pyversions/napari.svg) by
 using pip:
```

Also, suggest removing this link here. 

![Zrzut ekranu z 2022-07-17 22-13-06](https://user-images.githubusercontent.com/3826210/179423271-76decf7e-8705-48f6-b91f-0b1d836a177b.png)